pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -75,7 +75,7 @@ func (pl *CSILimits) Filter(ctx context.Context, _ *framework.CycleState, pod *v
 
 	node := nodeInfo.Node()
 	if node == nil {
-		return framework.NewStatus(framework.Error, fmt.Sprintf("node not found"))
+		return framework.NewStatus(framework.Error, "node not found")
```

Comment:
could these `Sprintfs` actually give some info from the `nodeInfo` about which node was not found?

Reply:
that should be better.

Reply:
Done